### ISSUE TITLE
Coiote's shadow composition

### ISSUE THREAD

**Issue by MLopezJ** [NordicSemiconductor/asset-tracker-cloud-coiote-aws-converter-js#24]
### TLDR

1. HOCON-based LwM2M object 
2. Custom Av System object
3. Not recognized objects in Coiote

## 1- HOCON-based LwM2M object 

Coiote's shadow is composed by their version of LwM2M objects, which is generated using a HOCON-based system and basically what it does is to replace `ids` of objects with their `name`, making the output human friendly.

``` json
{
"Temperature": {
  "0": {
    "Application Type": "",
    "Fractional Timestamp": {
      "noValue": true
    },
    "Max Measured Value": "23.51",
    "Max Range Value": "85.0",
    "Measurement Quality Indicator": {
      "noValue": true
    },
    "Measurement Quality Level": {
      "noValue": true
    },
    "Min Measured Value": "23.51",
    "Min Range Value": "-40.0",
    "Reset Min and Max Measured Values": {
      "noValue": true
    },
    "Sensor Units": "Celsius degrees",
    "Sensor Value": "24.57",
    "Timestamp": "2022-10-07T13:33:22Z"
    }
  }
}
```

However, in the shadow can be other escenarios

## 2- Custom Av System objects

``` json
"Location Assistance": {
  "0": {
    "Assistance type": "4",
    "A-GPS assistance mask": "128",
    "P-GPS predictionCount": "0",
    "P-GPS predictionIntervalMinutes": "0",
    "P-GPS startGpsDay": "0",
    "P-GPS startGpsTimeOfDaySeconds": "0",
    "accuracy": "526.0",
    "altitude": {
      "noValue": true
    },
    "assistance_data": {
      "noValue": true
    },
    "latitude": "63.42061758",
    "longitude": "10.43935061",
    "result code": {
      "noValue": true
    }
  }
}
```

## 3- Not recognized objects in Coiote

Last possible case scenario is when:

`
Coiote is missing proper object definition and has no translation for the key. 
`

``` json
{
  "33605": {},
  "33606": {
    "0": {
      "0": "Dummy_OBU_ID",
      "1": "PL 473N0",
      "2": "01-345678-901234",
      "3": "26000007",
      "4": "8926000000073",
      "5": "60.0",
      "6": "26001.0",
      "7": "GMSK",
      "8": "GMSK",
      "9": "1950.0",
      "10": "2140.0",
      "11": "6069.0",
      "12": "11503.0",
      "13": "0.0",
      "14": "0.0",
      "15": "4.5218974"
    }
  },
}

```
